packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -1140,6 +1140,25 @@ void main() {
     expect(inputBox.hitTest(BoxHitTestResult(), position: inputBox.globalToLocal(newFourthPos)), isFalse);
   });
 
+  testWidgets('minLines cannot be greater than maxLines', (WidgetTester tester) async {
+    try {
+      await tester.pumpWidget(
+        overlay(
+          child: Container(
+            width: 300.0,
+            child: SelectableText(
+              'abcd',
+              minLines: 4,
+              maxLines: 3,
+            ),
+          ),
+        ),
+      );
+    } on AssertionError catch (e) {
+      expect(e.toString(), contains("minLines can't be greater than maxLines"));
+    }
+  });
+
```

Comment:
Can you add a test to make sure minline actually works?
The test should be something as following. A really long SelectableText with minline equal to some small number(let's say two). verify the lines are actually two by checking the height of the RenderBox. ```tester.renderObject(find.byType(SelectableText)).size.height == <n pixel of two lines>```

Reply:
Edit: A really short SelectableText. I was thinking of maxline